docs/api-conventions.md

```diff
@@ -462,24 +462,24 @@ The status object is encoded as JSON and provided as the body of the response.
 **Example:**
 ```JSON
 >HTTP Requst:
-POST /api/v1beta1/events/ HTTP/1.1
-Authorization: Basic ...
+>POST /api/v1beta3/namespaces/default/events/ HTTP/1.1
```

Comment:
While you're at it, might as well use a better example:
```
> GET /api/v1beta3/namespaces/default/pods/grafana HTTP/1.1
 
< HTTP/1.1 404 Not Found
< Content-Type: application/json
< Date: Wed, 20 May 2015 00:21:46 GMT
< Content-Length: 232
 
{
  "kind": "Status",
  "apiVersion": "v1beta3",
  "metadata": {},
  "status": "Failure",
  "message": "pods \"grafana\" not found",
  "reason": "NotFound",
  "details": {
    "id": "grafana",
    "kind": "pods"
  },
  "code": 404
}
```
This shows the optional `details` field as well.